src/general/DifficultyPreset.cs

```diff
@@ -52,7 +52,7 @@ public class DifficultyPreset : IDifficulty, IRegistryType
     public bool PassiveReproduction { get; private set; }
 
     [JsonProperty]
-    public bool LimitGrowthRate { get; private set; }
+    public bool LimitGrowthRate { get; set; }
```

Comment:
Presets are registry data *they are not* allowed to be modified during runtime, otherwise starting new games will use incorrect data.